src/components/lab/PositiveIon.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Circle, Text, Group } from 'react-konva';
+import {
+  POSITIVE_ION_RADIUS,
+  POSITIVE_ION_COLOR,
+  POSITIVE_ION_SYMBOL,
+  POSITIVE_ION_SYMBOL_COLOR,
+  POSITIVE_ION_SYMBOL_FONT_SIZE,
+  POSITIVE_ION_SYMBOL_HORIZONTAL_ADJUSTMENT_FACTOR,
+  POSITIVE_ION_SYMBOL_VERTICAL_ADJUSTMENT_FACTOR,
+} from '../../config/constants';
+
+const PositiveIon = ({ xPosition, yPosition }) => {
+  return (
+    <Group>
+      <Circle
+        x={xPosition}
+        y={yPosition}
+        radius={POSITIVE_ION_RADIUS}
+        fill={POSITIVE_ION_COLOR}
+      />
+      <Text
```

Comment:
With these props I can have the symbol centered. There is a slight offset however, but I think it is better than having factors.
```

      <Text
        height={POSITIVE_ION_RADIUS * 2}
        width={POSITIVE_ION_RADIUS * 2}
        x={xPosition - POSITIVE_ION_RADIUS}
        y={yPosition - POSITIVE_ION_RADIUS}
        text={POSITIVE_ION_SYMBOL}
        fontSize={POSITIVE_ION_SYMBOL_FONT_SIZE}
        fill={POSITIVE_ION_SYMBOL_COLOR}
        align="center"
        verticalAlign="middle"
      />
```

Reply:
very nice, thank you. works as we change the radius as well